package.json

```diff
@@ -23,7 +23,7 @@
     "broccoli-merge-trees": "^1.0.0",
     "ember-cli-babel": "^6.6.0",
     "es5-shim": "~4.5.2",
-    "pixi.js": "~3.0.9"
+    "pixi.js": "~4.5.2"
```

Comment:
I still don't believe that that's a good pattern but if you say so...

shouldn't `pixi.js` be a `peerDependency` then if the user is supposed to choose a version?